services/disk/manager_disk.cpp

```diff
@@ -257,10 +257,11 @@ namespace services::disk {
         auto indexes = read_indexes_();
         metafile_indexes_->seek_eof();
         for (const auto& index : indexes) {
+            trace(log_, "manager_disk: load_indexes_ : {}", index.name());
             actor_zeta::send(dispatcher,
                              address(),
                              collection::handler_id(collection::route::create_index),
-                             session,
```

Comment:
if we pass current session to the create_index later, at the end of load index process this sesson will be removed, but create_index already referenced to it. When we use name for session this wasn't a problem, because find session searches by other key (session + name) which wasn't created yet.

Reply:
Here we need to add a comment when we change it so as not to lose it.

Reply:
Done